RetakesAllocator/RetakesAllocator.cs

```diff
@@ -72,7 +72,13 @@ public override void Load(bool hotReload)
         {
             HandleHotReload();
         }
+        RetakesAllocator retakesAllocatorInstance = new RetakesAllocator();
```

Comment:
Dont think we should do this

Reply:
i wanna separate on tick other events from original .cs to organize